tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -77,6 +77,7 @@
 #include <EnergyPlus/ElectricPowerServiceManager.hh>
 #include <EnergyPlus/FileSystem.hh>
 #include <EnergyPlus/General.hh>
+#include <EnergyPlus/GeneralRoutines.hh>
```

Comment:
Why is this header needed here?

Reply:
Earlier I tried a different unit test, which required calling a function in the `GeneralRoutines.cc`. It is not needed for the current unit test and should have been removed. I will do that shortly.